packages/cli/test/integration/shared/db/executions.ts

```diff
@@ -34,6 +39,16 @@ export async function createExecution(
 		deletedAt,
 	});
 
+	if (metadata?.length === 1) {
```

Comment:
When saving a new execution it's possible to store multiple key-value pairs as metadata, but the filtering currently only allows to filter by 1 key-value pair at a time.

I'm not sure why this changes was necessary and I don't think checking for `length === 1` would be a good choice either. 

Why was this code added here?

Reply:
But this is a call to create executions, that's why I'm confused

Reply:
Oh silly mixup, exact same line in two separate files and I thought you meant the other. Please see [`2286578`](https://github.com/n8n-io/n8n/pull/9477/commits/228657806bc758b164f89a007c08bb09c8727934).